code/Control/UserDefinedFormController.php

```diff
@@ -672,4 +678,20 @@ function (){
 
         return $result;
     }
+
+    /**
+     * Check validity of email and return array of valid emails
+     */
+    protected function validEmailsToArray(?string $emails): array
+    {
+        $emailsArray = [];
+        $emails = explode(',', $emails ?? '');
+        foreach ($emails as $email) {
+            if (filter_var($email = trim($email), FILTER_VALIDATE_EMAIL)) {
```

Comment:
I'm not doing a full review - just noticed this in passing
use [`Email::is_valid_address](https://api.silverstripe.org/5/SilverStripe/Control/Email/Email.html#method_is_valid_address) instead of `filter_var`

Reply:
Probably, I'll leave it as it is, since `'·broken@email'` also pass this test.

Reply:
If `symfony/mailer` accepts such sort of email, I'll do this change.